README.md

```diff
@@ -16,9 +16,6 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue)](https://github.com/feast-dev/feast/blob/master/LICENSE)
 [![GitHub Release](https://img.shields.io/github/v/release/feast-dev/feast.svg?style=flat&sort=semver&color=blue)](https://github.com/feast-dev/feast/releases)
 
-## Join us on Slack!
-👋👋👋 [Come say hi on Slack!](https://join.slack.com/t/feastopensource/signup)
```

Comment:
can you revert this

Reply:
Done.

Reply:
looks like it's added back.

Reply:
I think i had to remove it to make the pre-commit hook work. Since this is not my original change i have reverted as per Abdul request. Is this required or not required?

Reply:
It is. precommit issue was already fixed in #4423 btw, if you merge from master or also make the necessary change in [README.md.jinja2](https://github.com/feast-dev/feast/pull/4423/files#diff-cee58afa06956aae6cd0d7d63db2a7487441729e0281b5d46925bce80a7350a5) file, you should be fine.

Reply:
+1

Reply:
I see. Thanks for clarifying it. Now this PR branch is in sync with master. Please take a look at it now.